doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,81 @@
+# How to backport a Pull Request to a Release Line
+
+## Staging branches
+
+Each release line has a staging branch that the releaser will use a scratch pad
+while preparing a release. The branch name is formatted as follows:
+`vn.x-staging` where `n` is the major release number.
```

Comment:
Just a suggestion but how about a capital N? `vN.x-staging` might be a bit clearer.